vendor/github.com/containerd/continuity/fs/copy.go

```diff
@@ -72,7 +72,7 @@ func copyDirectory(dst, src string, inodes map[uint64]string) error {
 				if err := os.Link(link, target); err != nil {
 					return errors.Wrap(err, "failed to create hard link")
 				}
-			} else if err := copyFile(source, target); err != nil {
+			} else if err := CopyFile(target, source); err != nil {
```

Comment:
Why was the order of target and source switched?

Reply:
Ah; https://github.com/containerd/continuity/pull/106#discussion_r179883651

> We usually use LHS notation for the args in this package.

Confusing, as `CopyFileRange()` still uses `src`, `dst`